pom.xml

```diff
@@ -54,21 +54,21 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <!-- Java Version Dependencies -->
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
 
         <!-- Arquillian Dependencies -->
-        <version.arquillian>1.7.0.Alpha10</version.arquillian>
-        <payara.arquillian.container.version>3.0.alpha3</payara.arquillian.container.version>
```

Comment:
This is of course too optimistic, because of outstanding bug with translating deployment exceptions for the TCKs. I'll revert in those TCKs that are affected by it.